android/app/src/main/java/org/fossasia/openevent/data/Event.java

```diff
@@ -21,358 +16,311 @@
 public class Event extends RealmObject {
 
     @PrimaryKey
-    @Id(IntegerIdHandler.class)
-    private int id;
-    private String eventUrl;
-    private String description;
-    @JsonProperty("licence_details")
-    private LicenseDetails licenseDetails;
-    @JsonProperty("background_image")
-    private String backgroundImage;
-    private String startTime;
-    private String endTime;
-    private String ticketUrl;
-    private String topic;
-    private String thumbnail;
-    private String timezone;
-    private String logo;
-    private String searchableLocationName;
-    private String large;
-    private CallForPapers callForPapers;
-    private String locationName;
+    private String identifier;
     private String name;
-    private Copyright copyright;
-    private String privacy;
-    private String placeholderUrl;
-    @Relationship("social-links")
-    private RealmList<SocialLink> socialLinks;
+    private Double latitude;
```

Comment:
i am using the default ones we used earlier. will change it to double if you say.